pkg/api/server/register_exec.go

```diff
@@ -0,0 +1,330 @@
+package server
+
+import (
+	"github.com/containers/libpod/pkg/api/handlers"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+func (s *APIServer) registerExecHandlers(r *mux.Router) error {
+	// swagger:operation POST /containers/{name}/create compat createExec
+	// ---
+	// tags:
+	//   - exec (compat)
+	// summary: Create an exec instance
+	// description: Run a command inside a running container.
+	// parameters:
+	//  - in: path
+	//    name: name
+	//    type: string
+	//    required: true
+	//    description: name of container
+    //  - in: body
+    //    name: control
+    //    description: Attributes for create
+    //    schema:
```

Comment:
I'm probably going to have a version of this in libpod, FYI - we can likely use that for swagger generation

Reply:
Though, ours might have some extra stuff that we don't want in here, so I think we should keep this until we know